debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresChangeRecordEmitter.java

```diff
@@ -367,6 +367,13 @@ private Table tableFromFromMessage(List<ReplicationMessage.Column> columns, Tabl
                                     .nativeType(type.getRootType().getOid());
                             columnEditor.length(column.getTypeMetadata().getLength());
                             columnEditor.scale(column.getTypeMetadata().getScale());
+
+                            // as long as default value is not added to the decoded message metadata, we must apply
+                            // the current default read from the database
```

Comment:
As I mentioned on the other PR, I uncovered this small bug with one of my tests.

When streaming triggers a schema refresh, we will first read the table metadata from the database, then overwrite that with metadata from the message if `metadataInMessage == true`.

As the message metadata does not currently include the default values, this causes the default values to be overwritten with `null`. This would break the feature, so instead I propagate the most recent default value which was just read from the database.

I figured that adding this to the message metadata was well out of scope, if it would even be desired, but please let me know if there is any different way you would prefer to handle this issue.

This has the side-effect that default values may appear 'early' on the Connect Schema if multiple changes occur while the connector is stopped, though the real column values will still be correct. (Test documenting this is at L676 of PostgresConnectorIT)